challenge5/index.html

```diff
@@ -0,0 +1,292 @@
+            <h3 class="text-2xl font-bold">Emergency care</h3>
+            <p class="text-lg font-light text-gray-400">
+              You can get 24/7 urgent care for yourself or your children and
+              your lovely family
+            </p>
+          </div>
+        </div>
+
+        <div class="flex flex-col bg-primary_color md:w-1/4 px-8 rounded-2xl">
+          <div class="self-center py-5">
+            <img src="./assets/imgs/Frame-6.svg" alt="frame" />
+          </div>
+          <div class="flex flex-col gap-y-3 self-center pb-2">
+            <h3 class="text-2xl font-bold">Tracking</h3>
+            <p class="text-lg font-light text-gray-400">
+              Track and save your medical history and health data
+            </p>
+          </div>
+        </div>
+      </div>
+
+      <div class="flex self-center pt-15">
+        <button class="bg-primary_color text-secondary_color rounded-full text-button px-10 w-button border-solid border-2 border-secondary_color hover:opacity-80">
+          Learn more
+        </button>
+      </div>
+    </section>
+
+    <section class="relative flex flex-col md:w-3/4 pr-7 pl-5 py-20 self-center justify-center">
+      <img src="./assets/imgs/bgImg.svg" alt="Background Image" class="absolute right-0 transform -rotate-160 translate-x-2/3">
+      
+      <div class="flex flex-col pb-8 relative z-10">
+        <h2 class="text-3xl font-bold text-center pb-7">
+          Check out our latest article
+        </h2>
+        <div class="self-center">
+          <img src="./assets/icons/Rectangle.svg" alt="Vector" />
+        </div>
+      </div>
+
+      <div class="flex flex-col md:flex-row gap-y-9 md:gap-8 relative z-10">
+        <div class="flex flex-col rounded-2xl overflow-hidden bg-white">
+          <div class="w-full">
+            <img src="./assets/imgs/image.svg" alt="Illustration" class="w-full" />
+          </div>
+          <h3 class="text-2xl font-bold pt-6 pb-3 pl-8 pr-16">
+            Disease detection, check up in the laboratory
+          </h3>
+          <p class="text-lg pb-6 pl-8 pr-12 font-light text-gray-400">
+            In this case, the role of the health laboratory is very important
+            to do a disease detection...
+          </p>
+          <div class="flex flex-row mt-auto pl-8 pr-12 pb-8 hover:opacity-80">
+            <a href="#" class="text-secondary_color text-lg font-medium pr-3">Read more</a>
+            <img src="./assets/icons/arrow.svg" alt="Arrow" />
+          </div>
+        </div>
+
+        <div class="flex flex-col rounded-2xl overflow-hidden bg-white">
+          <div class="w-full">
+            <img src="./assets/imgs/image2.svg" alt="Illustration" class="w-full" />
+          </div>
+          <h3 class="text-2xl font-bold pt-6 pb-3 pl-8 pr-16">
+            Herbal medicines that are safe for consumption
+          </h3>
+          <p class="text-lg pb-6 pl-8 pr-12 font-light text-gray-400">
```

Comment:
Instead of using all these paddings here at the `h3` and `p`, we should set them at the container div (to avoid duplicating the paddings).

It is not very common to have paddings in `p` or `h` elements.

Also, it is not very common to have 4 different paddings (b, l, r and t) like we have here. I see in the design the padding on the right doesn't match the one on the left, but I think that's just because of the words not fitting in the line, and at least the padding on the left and the right should be the same.

If I'm not explaining myself well fell free to write me in slack!